letta-ai/letta
# feat: orm ToolsAgents migration

**Please describe the purpose of this pull request.**
Added ToolsAgents ORM with test cases

**How to test**
How can we test your PR during review? What commands should we run? What outcomes should we expect?
```
poetry run pytest -s -vv tests/test_managers.py
```

**Have you tested this PR?**
✅ 

**Related issues or PRs**
Please link any related GitHub [issues](https://github.com/cpacker/Letta/issues) or [PRs](https://github.com/cpacker/Letta/pulls).

**Is your PR over 500 lines of code?**
If so, please break up your PR into multiple smaller PRs so that we can review them quickly, or provide justification for its length.

**Additional context**
Add any other context or screenshots about the PR here.
